ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -526,19 +531,36 @@ def test_userroot_ldif_files_ownership(self):
         self.master.run_command(['ipa-restore', backup_path],
                                 stdin_text=dirman_password + '\nyes')
 
+        result = self.master.run_command(['umask'])
+        assert '0022' in result.stdout_text
+
```

Comment:
This test is not validating the fix. The proper scenario would be:
- set umask to 077
- call ipa-restore
- check that the output of ipa-restore displays 'Temporary setting umask to 022' and 'Restoring umask to 077'
- check that umask is 077